<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yoga</class>
 <widget class="QMainWindow" name="yoga">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1478</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 1,
       stop: 0 #f7c6d9,  
       stop: 1 #e6e6fa
    );
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="block1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>411</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #ff992;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="6" column="0">
      <widget class="QLabel" name="Pose1">
       <property name="styleSheet">
        <string notr="true">QLabel{
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;
}
</string>
       </property>
       <property name="text">
        <string>Childâ€™s Pose </string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;


</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="yogarcs.qrc">:/new/prefix1/image/balasana.jpeg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="button1">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #aa1316;    
    color: white;                   
    font: bold 14px &quot;Arial&quot;;
    border-radius: 25px;
    padding: 6px 6px;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditFind"/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QTextBrowser" name="text1">
       <property name="styleSheet">
        <string notr="true">QTextBrowser{
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;
}

</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:20px; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;(Balasana)&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Duration: 1-3 minutes &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://youtu.be/2MJGg-dUKh0?si=w9LfUnxrCke-sXWZ&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Video for Child's Pose&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="block2_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>491</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #ff992;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;</string>
    </property>
    <layout class="QGridLayout" name="block2">
     <item row="0" column="0">
      <widget class="QLabel" name="Pose2">
       <property name="styleSheet">
        <string notr="true">QLabel{
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;
}

</string>
       </property>
       <property name="text">
        <string>Reclined Bound Angle Pose </string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;


</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="yogarcs.qrc">:/new/prefix1/image/reclinedbound.jpeg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="button2">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #aa1316;    
    color: white;                   
    font: bold 14px &quot;Arial&quot;;
    border-radius: 25px;
    padding: 6px 6px;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditFind"/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QTextBrowser" name="text2">
       <property name="styleSheet">
        <string notr="true">QTextBrowser{
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;
}

</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:20px; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;(Supta Baddha Konasana)&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Duration: 3-5 minutes&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://youtu.be/Z1G8rM-0w2U?si=i2iSZ4VW_JYumEQB&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Video for Reclined Pose&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="block4_2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>460</y>
      <width>501</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="block4">
     <item row="0" column="2">
      <widget class="QPushButton" name="button4">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #aa1316;    
    color: white;                   
    font: bold 14px &quot;Arial&quot;;
    border-radius: 25px;
    padding: 6px 6px;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditFind"/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="Pose4">
       <property name="styleSheet">
        <string notr="true">QLabel{
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;
}


</string>
       </property>
       <property name="text">
        <string> Bridge Pose </string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;



</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="yogarcs.qrc">:/new/prefix1/image/bridge.jpeg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QTextBrowser" name="text4">
       <property name="styleSheet">
        <string notr="true">QTextBrowser{
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;
}

</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:20px; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;(Setu Bandha Sarvangasana)&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Duration: 30 seconds to 1 minute &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://youtu.be/XUcAuYd7VU0?si=lQwjQ5TDc5qxU76w&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Video for Bridge Pose&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="block5_3">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>150</y>
      <width>501</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="block5">
     <item row="0" column="2">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;


</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::RichText</enum>
       </property>
       <property name="pixmap">
        <pixmap resource="yogarcs.qrc">:/new/prefix1/image/Cat-Cow-Pose.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="button5">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #aa1316;    
    color: white;                   
    font: bold 14px &quot;Arial&quot;;
    border-radius: 25px;
    padding: 6px 6px;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditFind"/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="Pose5">
       <property name="styleSheet">
        <string notr="true">QLabel{
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;
}


</string>
       </property>
       <property name="text">
        <string>Cat-Cow Stretch</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="2" colspan="3">
      <widget class="QTextBrowser" name="text5">
       <property name="styleSheet">
        <string notr="true">QTextBrowser{
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;
}

</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:20px; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;(Chakravakasana)&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Duration: 1-2 minutes &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; &lt;a href=&quot;https://youtu.be/vuyUwtHl694?si=1yZ2jVuOuAHsEBcW&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Video for Cat-Cow stretch&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="block6_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>460</y>
      <width>491</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="block6">
     <item row="1" column="2">
      <widget class="QPushButton" name="button6">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #aa1316; 
    color: white;                   
    font: bold 14px &quot;Arial&quot;;
    border-radius: 25px;
    padding: 6px 6px;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditFind"/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Pose6">
       <property name="styleSheet">
        <string notr="true">QLabel{
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;
}

</string>
       </property>
       <property name="text">
        <string>Legs-Up-The-Wall Pose</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;


</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="yogarcs.qrc">:/new/prefix1/image/legup.jpeg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QTextBrowser" name="text6">
       <property name="styleSheet">
        <string notr="true">QTextBrowser{
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;
}

</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:20px; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;(Viparita Karani)&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Duration: 3-5 minutes &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://youtu.be/do_1LisFah0?si=bz6ni4YR-hC5A2yh&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Video for Legs Up the Wall pose&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="yogarcs.qrc">:/image/1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Heading">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>489</width>
      <height>78</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;
}


</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:400;&quot;&gt;YOGA POSES FOR PERIODS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="block3_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>411</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="block3">
     <item row="2" column="3">
      <widget class="QPushButton" name="button3">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #aa1316; 
    color: white;                   
    font: bold 14px &quot;Arial&quot;;
    border-radius: 25px;
    padding: 6px 6px;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditFind"/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Pose3">
       <property name="styleSheet">
        <string notr="true">QLabel{
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;
}

</string>
       </property>
       <property name="text">
        <string>Seated Forward Bend</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;


</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="yogarcs.qrc">:/new/prefix1/image/Paschimottanasana-Seated-Forward.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QTextBrowser" name="text3">
       <property name="styleSheet">
        <string notr="true">QTextBrowser{
    background-color: #F497B6;      
    color: #3D3D3D;                   
    font: bold 20px &quot;Arial&quot;;
    border-radius: 8px;
    padding: 6px 12px;
}

</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:20px; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;(Paschimottanasana)&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Duration: 1-3 minutes &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; &lt;a href=&quot;https://youtu.be/SLIaql7h6RQ?si=wso-DU_unF7uUVhe&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Video for Seated Forward Pose&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="yogarcs.qrc"/>
 </resources>
 <connections/>
</ui>
